python/ray/ml/tests/test_tensorflow_predictor.py

```diff
@@ -43,7 +43,7 @@ def test_init():
     assert checkpoint_predictor.preprocessor == predictor.preprocessor
```

Comment:
Out-of-scope for this PR, but we should add tests for dataframes. Related to #25193.